content_scripts/setupMenuBar.js

```diff
@@ -34,6 +34,7 @@ import Splatting from '../../src/splatting/Splatting.js';
         DebugAvatarConnections: 'Debug Avatar Connections',
         DeleteAllTools: 'Delete All Tools',
         DownloadScan: 'Download Scan',
+        DownloadZipBackup: 'Download World Object',
```

Comment:
nit: unrelated change crept in from #201

Reply:
whoops